README.md

```
@@ -51,6 +37,36 @@ $ ember generate fragment foo someAttr:string anotherAttr:boolean
+
+Create or update your application serializer to extend one of the fragment-aware serializers:
+
+```javascript
+// app/serializers/application.js
+
+import FragmentSerializer from "ember-data-model-fragments/serializer";
+
+export default class ApplicationSerializer extends FragmentSerializer {}
+```
+
+See the [Serialization](#serialization) section for more options if you're using `RESTSerializer` or `JSONAPISerializer`.
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)